app/models/chargeback_rate_detail.rb

```diff
@@ -16,11 +16,18 @@ def find_rate(value)
     fixed_rate = 0.0
     variable_rate = 0.0
     chargeback_tiers.each do |tier|
-      next if value < rate_adjustment(tier.start)
-      next if value >= rate_adjustment(tier.finish)
-      fixed_rate = tier.fixed_rate
-      variable_rate = tier.variable_rate
-      break
+      if tier.starts_with_zero?
```

Comment:
@tledesma oh I see, but it is not so much understandable but you also include this case to select method, 
because I believe that it is for cases where value is exactly 0
for example:

```
    tier_found, = chargeback_tiers.select do |tier|
      tier.starts_with_zero? && value.zero? ||
      value > rate_adjustment(tier.start) && value <= rate_adjustment(tier.finish)
    end
```

Is it covering all case what do have on mind ?
Is it more readable for you also ?

thanks!